packages/api-types/src/dataviews.ts

```diff
@@ -1,8 +1,8 @@
 import { ApiAppName, Dataset } from '.'
 
-export interface DataviewConfig<T = any> {
+export interface DataviewConfig<Type = any> {
   // TODO use any property from layer-composer here?
-  type?: T
+  type?: Type
```

Comment:
thanks, now it's more clear.

Reply:
no, thanks to you! I got it from the typescript errors to avoid link you sent